### ISSUE TITLE
Error when launching main.py in a virtualenv

### ISSUE THREAD

**Issue by tok43** [risingfbk/fluidos-energy-predictor#2]
When launching the program with the following command: `python3 src/main.py` as specified in the documentation I got the below import error from python:

```
(energy-predictor) ➜  fluidos-energy-predictor git:(review) python3 src/main.py
Traceback (most recent call last):
  File "/PATH/TO/REPO/fluidos-energy-predictor/src/main.py", line 9, in <module>
    import src.model as modelmd
ModuleNotFoundError: No module named 'src'
```

**Comment by mfranzil** [comment]
Fixed as of `v0.2.3`. Please check

**Comment by tok43** [comment]
I confirm this is working now
